src/test/java/com/bio4j/angulillos/TwitterGraphTestSuite.java

```diff
@@ -14,8 +14,9 @@
 
   public TwitterSchema<RV,RE>.User.Vertex addUser(String name, Integer age) {
 
-    return g.addVertex(g.user)
-            .set(g.user.name, name)
+    TwitterSchema<RV,RE>.User.Vertex u = g.user.addVertex();
```

Comment:
yes, but then how are we gettting the right type? through calling the right constructor *once* an schema has been implemented?